### ISSUE TITLE
Insufficient contrasts detected by Pa11y test

### ISSUE THREAD

**Issue by JakeChampion** [Financial-Times/o-forms#165]
To replicate this you can run `obt install && obt demo && obt test`.

```
Executing Pa11y
    →
    →  • Error: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text
 in this element has a contrast ratio of 4.34:1. Recommendation: change text colour to #0a737d.
    →    ├── WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    →    ├── html > body > div:nth-child(29) > div > div > button
    →    └── <button type="button" class="o-buttons">Go</button>
    →
    →  • Error: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text
 in this element has a contrast ratio of 4.34:1. Recommendation: change text colour to #0a737d.
    →    ├── WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    →    ├── html > body > div:nth-child(30) > div > div > button
    →    └── <button type="button" class="o-buttons">Go</button>
    →
    →  • Error: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text
 in this element has a contrast ratio of 4.34:1. Recommendation: change text colour to #0a737d.
    →    ├── WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    →    ├── html > body > div:nth-child(31) > div > div > button
    →    └── <button type="button" class="o-buttons">Go</button>
    →
    →  • Error: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text
 in this element has a contrast ratio of 4.34:1. Recommendation: change text colour to #0a737d.
    →    ├── WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    →    ├── html > body > div:nth-child(32) > div > div > button
    →    └── <button type="button" class="o-buttons">Go</button>
    →
    →  • Error: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text
 in this element has a contrast ratio of 4.34:1. Recommendation: change text colour to #0a737d.
    →    ├── WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    →    ├── html > body > div:nth-child(36) > div > div > button
    →    └── <button type="button" class="o-buttons">Go</button>
    → Failed Pa11y tests
```

**Comment by alicebartlett** [comment]
This design has changed in #164 and that should remove this as a problem.

**Comment by notlee** [comment]
👍 this has been released as part of v5 https://github.com/Financial-Times/o-forms/pull/180
